examples/sphere/shallow_water.jl

```diff
@@ -77,6 +91,17 @@ elseif test_name == steady_state_compact_test_name
     const ϕᵦ = -30.0
     const ϕₑ = 90.0
     const xₑ = 0.3
+elseif test_name == barotropic_instability_test_name
+    const u_max = 80.0
+    const xₑ = 0.3
+    const αₚ = 19.09859
+    const βₚ = 3.81971
+    const h0 = 10158.18617 # value for initial height from Tempest https://github.com/paullric/tempestmodel/blob/master/test/shallowwater_sphere/BarotropicInstabilityTest.cpp#L86
+    const h_hat = 120.0
+    const ϕ₀ = 25.71428
+    const ϕ₁ = 64.28571
+    const ϕ₂ = 45.0
+    const eₙ = exp(-4.0 / (π * ϕ₁ / 180.0 - π * ϕ₀ / 180.0)^2)
```

Comment:
I think it would be clearer to use the function `deg2rad()` consistently doing the conversions in this example.

Reply:
Okay, I can do that!